instagram-app/src/components/CommentSection.css

```diff
@@ -0,0 +1,19 @@
+form{
+    display: flex;
+}
+input{
+flex: 8;
+border: none;
+border-bottom: 2px solid black;
```

Comment:
Although there's no nesting with vanilla CSS, I'd still recommend indenting property settings to visually distinguish them from selectors.